amethyst_assets/src/loader.rs

```diff
@@ -69,10 +71,9 @@ impl<A, E> Future for AssetFuture<A, E> {
                 // reference gets dropped and we enter this branch.
                 // Thus, we are the only one with access to the cell.
                 unsafe {
-                    x.value
-                        .into_inner()
-                        .expect("Thread panicked")
-                        .map(Async::Ready)
+                    x.value.into_inner().expect("Thread panicked").map(
+                        Async::Ready,
```

Comment:
This formatting looks really weird to me because it does not clearly show that we're passing a function.